en/components/categorychart_configuring.md

```diff
@@ -59,14 +59,13 @@ This feature allows you to animate the series as it loads a new data source. The
 
 The chart resizes allowing for the title and subtitle information.
 
->[!NOTE] 
->By default, animated transitions are disabled and you can enable transition-in animations by setting the `isTransitionInEnabled` option to “true”.
+By default, animated transitions are disabled and you can enable transition-in animations by setting the `isTransitionInEnabled` option to “true”.
```

Comment:
I'd say transition in animations or intro animations here. To better distinguish from the other transitions that happen when data changes.